activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Set polymorphic type column to NULL on `dependent: :nullify` strategy. 
+    
+    On polymorphic associations both the `foreign key` and the `foreign type` columns will be set to NULL.
```

Comment:
Backticks are not needed since "foreign key" and "foreign type" are not a code block.

Reply:
@kamipo usually I put backticks to emphasise something but sure will change it tnx.